FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -986,36 +1012,36 @@ int Maslow_::get_direction(double x, double y, double targetX, double targetY){
     return direction;
 } 
 
-// Generate calibration points based on dimentions (TODO)
+// Generate calibration points based on dimentions
 void Maslow_::generate_calibration_grid() {
 
-    int gridSizeX = int(sqrt(CALIBRATION_GRID_SIZE));
-    int gridSizeY = int(sqrt(CALIBRATION_GRID_SIZE));
-    int xSpacing =  (frame_width - 2*calibration_grid_offset) / gridSizeX;
-    int ySpacing =  (frame_height - 2*calibration_grid_offset) / gridSizeY;
-    int pointCount = 0;
-
-  for(int i = -gridSizeX/2; i <= gridSizeX/2; i++) {
-    if(i % 2 == 0) {
-      for(int j = -gridSizeY/2; j <= gridSizeY/2; j++) {
-        log_info("Point: " << pointCount << "(" << i * xSpacing << ", " << j * ySpacing << ")");
-        calibrationGrid[pointCount][0] = i * xSpacing;
-        calibrationGrid[pointCount][1] = j * ySpacing;
-        pointCount++;
-      }
-    } else {
-      for(int j = gridSizeY/2; j >= -gridSizeY/2; j--) {
-        log_info("Point: " << pointCount << "(" << i * xSpacing << ", " << j * ySpacing << ")"); 
-        calibrationGrid[pointCount][0] = i * xSpacing;
-        calibrationGrid[pointCount][1] = j * ySpacing;
-        pointCount++;
-      }
-    }
-  }
+    int gridSizeX  = int(sqrt(CALIBRATION_GRID_SIZE));  //25 --> 5
```

Comment:
How can I set this back to 100 points? I think it works better for two reasons 

1. The movements are smaller so we have to worry about extra slack in the slacking off belts less 
2. Its the only calibration system that we've actually tested with cutting things so we know it works well

Reply:
Really 98 points or whatever

Reply:
Temporarily went back to fixed grid, because some grids didn't work in horizontal, for some belts needed to extend during move. Probably is fixed now, since I split the first diagonal move into two parts (pure X and Y movements).

Reply:
What would I need to do to go back to the larger grid?